src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -232,11 +242,12 @@
   ([raw-config :- {:jruby-puppet {schema/Keyword schema/Any}
                    (schema/optional-key :http-client) {schema/Keyword schema/Any}
                    schema/Keyword schema/Any}]
-   (initialize-and-create-jruby-config raw-config nil (fn []) false))
+   (initialize-and-create-jruby-config raw-config nil (fn []) false nil))
```

Comment:
Yes to both of you. @camlow325 this arity is only used by the CLI tools, and I think it's fine that they won't get a metric registry. @jpinsonault yeah, these will always be set to nil. I could change it so that they're only set if they are set to something in the map? Either way will work - that might be a little less confusing?